src/rascal/models/kernels.hh

```diff
@@ -346,7 +357,8 @@ namespace rascal {
         return this->compute_helper<Property_t, TargetType::Atom>(
             representation_name, managers_a, managers_b);
       default:
-        throw std::logic_error("The combination of parameter is not handled.");
+        throw std::logic_error(
```

Comment:
Seriously, is this condition ever reached? Even if it is, it's not a useful error message; I would just leave out the default case entirely.

Reply:
the compiler does not allow to not have a default in this case so I improved the message

Reply:
You shouldn't have bothered with the message; in fact, I think it's doing more harm than good by making a user think it's something _they_ did with some weird parameter combination, rather than being caused by a programmer error. If C++ allows throwing exceptions without a message, I would have just done that. Or at least write a message that makes it clear the programmer, not the user, is at fault.